README.md

```diff
@@ -22,6 +22,9 @@ Official binary sources (provided by davidgfnet):
 Building
 --------
 
+Dependencies:
+Libc6-dev, libfreeimage-dev
```

Comment:
Hahahaha do they? Are Debian users retarded? :P Yeah lots of Ubuntu folks, but come on, they do have a launchpad repo... Let them suffer a bit